packages/palette/src/elements/StackableBorderBox/StackableBorderBox.tsx

```diff
@@ -8,7 +8,7 @@ import { BorderBoxProps } from "../BorderBox/BorderBoxBase"
  * A stackable border box is a BorderBox that shares borders with its siblings.
  */
 export const StackableBorderBox = styled(BorderBox)<BorderBoxProps>`
-  padding: ${space(3)}px;
+  padding: ${space(4)}px;
```

Comment:
Ugh

Reply:
this comes from https://github.com/artsy/eigen/blob/main/src/palette/Theme.tsx#L246, where the designers said to take 3->4. but im cool either way.